testapps/TestLayerAspNetCore/TestLayerAspNetCore.csproj

```diff
@@ -5,7 +5,7 @@
     <AWSProjectType>Lambda</AWSProjectType>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
+    <PackageReference Include="Microsoft.AspNetCore.App" Version="2.1.11" />
```

Comment:
You can remove the `Version` attribute. We no longer do validation on that.

Reply:
removed